docs/using.md

```diff
@@ -107,11 +107,58 @@ steps:
         value: "world"
 ```
 
+### Resource shared between tasks
+
 Pipeline Tasks are allowed to pass resources from previous tasks via
 `providedBy` field. This feature is implemented using Persistent Volume Claim
 under the hood but however has an implication that tasks cannot have any volume
 mounted under path `/pvc`.
 
+### Outputs
+
+`Task` definition can include inputs and outputs resource declaration. If specific set of resources are only declared in output then a copy of resource to be uploaded or shared for next task is expected to be present under the path `/workspace/output/resource_name/`. 
+
+```yaml
+resources:
+  outputs:
+    name: storage-gcs
+steps:
+  - image: objectuser/run-java-jar #https://hub.docker.com/r/objectuser/run-java-jar/
+    command: [jar]
+    args: ["-cvf", "-o", "/workspace/outputs/storage-gcs/", "projectname.war", "*"]
```

Comment:
the path should be `/workspace/output/storage-gcs`

Reply:
ah. Good catch